doc/thirdPartyRunPlugins.md

```diff
@@ -38,3 +38,4 @@ Contact the developers of a plugin directly for assistance with a specific plugi
 | [Input Typer](https://github.com/CoreyHayward/PowerToys-Run-InputTyper) | [CoreyHayward](https://github.com/CoreyHayward) | Type the input as if sent from a keyboard |
 | [Clipboard Manager](https://github.com/CoreyHayward/PowerToys-Run-ClipboardManager) | [CoreyHayward](https://github.com/CoreyHayward) | Search and paste text from your clipboard history |
 | [Currency Converter](https://github.com/Advaith3600/PowerToys-Run-Currency-Converter) | [advaith3600](https://github.com/advaith3600) | Convert real and crypto currencies |
+| [1Password](https://github.com/KairuDeibisu/PowerToysRunPlugin1Password) | [KairuDeibisu](https://github.com/KairuDeibisu) | Search 1Password for usernames and passwords |
```

Comment:
I don't think this should be called 1Password as it isn't their plugin.  We can't imply it is official as it is not.

Reply:
What would you recommend?

I'm thinking,  

- PowerSearch for OnePassword

The name is fairly generic, and one password uses numerical 1 in all of their branding.

I'll also edit the description to explicitly state that the plugin is not affiliated or supported by 1password.

Reply:
> I don't think this should be called 1Password as it isn't their plugin. We can't imply it is official as it is not.

I'd like to highlight a discrepancy in how we're handling plugin attributions. For instance, we have a plugin named after Spotify, which isn't managed by Spotify itself but rather by a company named CDID Net. Despite this, it's been allowed and not questioned for implying an official connection.

Reply:
Great callout, https://github.com/microsoft/PowerToys/pull/31863 will address that to a degree

With things like passwords and auth, i want end users to be very critical of who has access to them and be eyes wide open. I still would very much like the name to clearly state this plugin is by not by 1Password even after #31863 is checked in.

Reply:
Hi @KairuDeibisu , Can you please merge latest main here and address the requests from crutkas? Thank you 😄